.github/workflows/publish-example-images.yaml

```diff
@@ -52,7 +52,9 @@ jobs:
           - component-name: mxnet-auto-tuning
             dockerfile: examples/mxnet/tune/Dockerfile
             context: examples/mxnet/tune
-
+          - component-name: train-api-hf-image
+            dockerfile: sdk/python/kubeflow/trainer/hf_dockerfile
+            context: sdk/python/kubeflow/trainer
```

Comment:
As we discussed in another thread, the `train-api-hf-image` is a component, not an image, for the examples.
So can we move this workflow to `publish-core-images`? https://github.com/kubeflow/training-operator/blob/ed3168a89a792f4b7dc6ac84ff5f9cb5f1a2469c/.github/workflows/publish-core-images.yaml

And then can we rename `train-api-hf-image` with `train-api-hf` or the `finetuning-huggingface-trainer`? We used not to add a suffix, `image`, to the image name.

@andreyvelich @johnugeorge WDYT of image name? Any better suggestions?

Reply:
I agree with you, let's move it to the `publish-core-images` workflow and name it as 
`trainer-huggingface`. So the final image will be `docker.io/kubeflow/trainer-huggingface`

Reply:
+1 
`trainer-huggingface` looks good

Reply:
+1 on `trainer-huggingface`.